test/unit/meta/src/CudaVectorArrayWrapperTest.cpp

```diff
@@ -37,46 +75,25 @@ class CudaVectorArrayWrapperTestKernel
         static_assert(T1::size == 1, "CudaVectorArrayWrapper in-kernel size test failed!");
         static_assert(std::tuple_size<T1>::value == 1, "CudaVectorArrayWrapper in-kernel tuple_size test failed!");
         static_assert(std::is_same<decltype(t1[0]), T&>::value, "CudaVectorArrayWrapper in-kernel type test failed!");
-#    ifdef __GNUC__
-#        pragma GCC diagnostic push
-#        pragma GCC diagnostic ignored "-Wfloat-equal"
-#    endif
-        ALPAKA_CHECK(*success, t1[0] == 0);
-#    ifdef __GNUC__
-#        pragma GCC diagnostic pop
-#    endif
+        ALPAKA_CHECK(*success, equals(t1[0], 0));
```

Comment:
Why not:
```suggestion
        ALPAKA_CHECK(*success, equals(t1[0], T{0}));
```
?

Reply:
Yes. I don't know how I forgot about this. I must have been mentally stuck somewhere in C++11 for some reason.